### DISCUSSION TITLE
CPU issue while trying to run gnb on a ubuntu virtual machine

### DISCUSSION THREAD

**Discussion by AmatipIT** [srsran/srsRAN_Project#179] [General Help]
Can anyone help with possible solution to this error

VirtualBox:~/srsLTE/build$ gnb
srsLog error - Unable to create log file "/tmp/gnb.log": Permission denied
srsGNB ERROR: The CPU does not support the required CPU features that were configured during compile time: sse4.1(ok) avx(ok) avx2(ok) fma(na) pclmul(ok)

Aborted (core dumped)
ola@ola-VirtualBox:~/srsLTE/build$

**Comment by pgawlowicz** [comment]
please run as:
```
sudo ./gnb -c ./gnb.yaml
```

Please note that a virtual CPU does not support the required CPU features. Please build the project again with correct cmake config.

**Reply by azzukky** [reply]
Please can you direct me to the correct cmake config to make this work?

**Reply by pgawlowicz** [reply]
The default cmake file is here. It should automatically detect the available ISA:
https://github.com/srsran/srsRAN_Project/blob/main/CMakeLists.txt#L69
